token/services/db/sql/tokens.go

```diff
@@ -884,43 +837,13 @@ func (db *TokenDB) GetSchema() string {
 			is_deleted BOOL NOT NULL DEFAULT false,
 			spent_by TEXT NOT NULL DEFAULT '',
 			spent_at TIMESTAMP,
+			owner BOOL NOT NULL DEFAULT false,
+			auditor BOOL NOT NULL DEFAULT false,
+			issuer BOOL NOT NULL DEFAULT false,
 			PRIMARY KEY (tx_id, idx)
 		);
 		CREATE INDEX IF NOT EXISTS idx_spent_%s ON %s ( is_deleted );
```

Comment:
Maybe you want to include owner in the index.

Also you may want to try CREATE BITMAP INDEX instead of index. In theory it should work well with such cases.